### ISSUE TITLE
Should i use botframework?

### ISSUE THREAD

**Issue by Nisthar** [ShyykoSerhiy/skyweb#61]
Hi,

I checked out botframework. Its good.

But its hard to get pass their review process (it takes a long time). I want to know if skyweb supports all the features of botframework? If not, which features will be skyweb missing.

Sorry for bad english.

**Comment by ShyykoSerhiy** [comment]
Hi, 
If you are building production ready bot for consumer use, i would stick with botframework. Couple of reasons:

1. Official support and some kind of guarantees that it will not stop working in any moment.
2. It gives you possibility to connect to other platforms(like kik, slack, messanger, etc) "with the flip of a switch".

skyweb works through a group of hacks really. It's supposed to be used in hobbyist fun projects like hubot connectors or smth like that. Literally any second it may stop working and it has already happen before https://github.com/ShyykoSerhiy/skyweb/issues/59. 

So in conclusion: it depends on what you are planning to do.

**Comment by Nisthar** [comment]
The only problem i am facing with botframework is their reviewing.
